pages/migration/wsc-53/migration_wsc-53_libraries.md

```diff
@@ -12,3 +12,16 @@ No breaking changes are expected for simple uses.
 A detailed [Guzzle migration guide](https://github.com/guzzle/guzzle/blob/master/UPGRADING.md#60-to-70) can be found in the Guzzle documentation.
 
 The explicit `sink` that was recommended in the [migration guide for WSC 5.2](migration_wsc-52_libraries.html#guzzle) can now be removed, as [the Guzzle issue #2735](https://github.com/guzzle/guzzle/issues/2735) was fixed in Guzzle 7.
+
+## Emogrifier / CSS Inliner
+
+The Emogrifier library was updated from version 2.2 to 5.0.
+Version 3.0 deprecated the `Emogrifier` class in favor of the `CssInliner` class that was added in 2.1.
+The `Emogrifier` class was subsequently removed in version 4.0.
```

Comment:
May-be just state to use `CssInliner` instead of `Emogrifier`. The exact history of when the changes appear to irrelevant here.

Reply:
I considered that important, because the Emogrifier class was removed without deprecating it within our software first, because they released so quickly that we missed several versions.

Reply:
I understand but the details of when it happened is irrelevant for our documentation, in my opinion. Just stating that the `Emogrifier` class was removed and that the `CssInliner` class has to be used instead should suffice.

Reply:
I've rephrased that paragraph, PTAL.